index.html

```diff
@@ -0,0 +1,260 @@
+                        <p class="price__free">Free for life </p>
+                        <div class="about__price">1 GB of space <hr>1 GB of space <hr>1 GB of space <hr>1 GB of space <hr>1 GB of space <hr>1 GB of space <hr>1 GB of space </div>
+                    </div>
+                    <div class="price__content">
+                        <p class="price__title">Pro <hr class="hr_price_white"></p>
+                        <h3 class="price__sum ">$30 <hr class="hr_price_white act"></h3>
+                        <p class="price__free">Free for life </p>
+                        <div class="about__price">1 GB of space <hr>1 GB of space <hr>1 GB of space <hr>1 GB of space <hr>1 GB of space <hr>1 GB of space <hr>1 GB of space </div>
+                    </div>
+          </div>
+      </div>
+  </section>
+  <section class="customers">
+      <div class="wrapper__price act">
+          <h2 class="title_customer">what our customers are saying</h2>
+          <hr class="price_hr">
+          <p class="text_customer">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+            <div class="customer__container">
+                <div class="customer__content">
+                        <div class="info_customer">Lorem ipsum dolor sit amet, 
+                            consectetur adipisicing elit. Doloribus 
+                            accusamus expedita repellat similique 
+                            odio aspernatur ex, architecto eaque 
+                            quo suscipit.</div>
+                        <div class="info_customer act">Lorem ipsum dolor sit amet, 
+                            consectetur adipisicing elit. Doloribus 
+                            accusamus expedita repellat similique 
+                            odio aspernatur ex, architecto eaque 
+                            quo suscipit.</div>
+                        <div class="info_customer">Lorem ipsum dolor sit amet, 
+                            consectetur adipisicing elit. Doloribus 
+                            accusamus expedita repellat similique 
+                            odio aspernatur ex, architecto eaque 
+                            quo suscipit.</div>    
+                </div>
+                <div class="about__customer">
+                    <div class="ava__customer">
+                        <img src="./image/Ellipse7.png" alt="ava">
+                    </div>
+                    <p class="name_customer">Davis Rhiel Madsen <br> <span>Manager</span> </p>
+                    <div class="ava__customer">
+                        <img src="./image/Ellipse8.png" alt="ava">
+                    </div>
+                    <p class="name_customer">Jaydon Rosser <br> <span>Manager</span> </p>
+                    <div class="ava__customer">
+                        <img src="./image/Ellipse9.png" alt="ava">
+                    </div>
+                    <p class="name_customer">Cooper Dorwart <br> <span>Manager</span> </p>
```

Comment:
class customer, naming I don't understand. If should write like customer__element

Reply:
done